README.md

```diff
@@ -25,7 +29,95 @@ If neither credentials file and env vars are found Swing will fail.
 To create your access keys see the [official AWS documentation][aws-credentials-docs].
 To know more about the environment variables see [this other documentation][aws-env-vars-docs].
 
-# Necessary AWS IAM permissions
+## Configuration Examples
+
+### Using AWS Client
+
+Install aws-cli using your favourite package manager or windows installer and Run
+
+```
+$ aws configure
+AWS Access Key ID [None]: my-aws-key
+AWS Secret Access Key [None]: my-aws-secret-key
+Default region name [None]: 
+Default output format [None]:
+```
+
+### Using Environment Variables
+
+***Linux***
+
+```
+esport AWS_ACCESS_KEY_ID=my-aws-key 
+export AWS_SECRET_ACCESS_KEY=my-aws-secret-key 
+```
+
+For a persistent configuration copy above lines in your
+
+```~/.bashrc``` or ```~/.zshrc``` depending on your shell 
+
+***Windows***
+
+TODO
+
+# Users
+
+## Quickstart
+
+### Existing projects using swing
+
+On Existing projects it is supposed to have swing.csv somewhere in the code. if it's in the same dir swing is you'll just run
+
+```
+$ swing -d -s pathto/swing.csv
+```
+
+### New projects 
+
+Just start uploading your files on an existing S3 bucket. You must have access to the bucket specified with option `-bucket` (Check the following Usage Section for details)
```

Comment:
Shouldn't the correct flag be `--bucket`?

Reply:
not, options are single hyphen

Reply:
it would need a change also in the code I guess

Reply:
Yeah, you're right. The printed usage section is wrong then, it should be updated in another PR.